src/libraries/tests.proj

```diff
@@ -46,6 +46,11 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Runtime.InteropServices\tests\DllImportGenerator.Tests\DllImportGenerator.Tests.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetArchitecture)' == 'armel'">
+    <!-- DllImportGenerator runtime tests depend on DNNE, which does not support armel as we don't officially support it. -->
```

Comment:
What is "DNNE"?

Reply:
https://github.com/AaronRobinsonMSFT/DNNE, used as a dependency for `GeneratedDllImportAttribute` tests.